pkg/cloud/ibmcloud/deployer.go

```diff
@@ -89,3 +92,34 @@ func (d *DeploymentClient) GetKubeConfig(cluster *clusterv1.Cluster, master *clu
 	}
 	return strings.TrimSpace(parts[1]), nil
 }
+
+func getSSHKeyFile(homeDir string) string {
+	defaultKey := homeDir + "/.ssh/id_ibmcloud"
+	envPrivateKey, ok := os.LookupEnv("IBMCLOUD_HOST_SSH_PRIVATE_FILE")
+	if !ok || (0 == strings.Compare(envPrivateKey, "")) {
+		return defaultKey
+	}
+	klog.V(3).Infof("Found environment variable SSH private file %s ", envPrivateKey)
+	exist, _ := FileExists(envPrivateKey)
```

Comment:
The wrapper function makes less sense for this case.

Reply:
User will ensure the ssh key file path and update code.